### DISCUSSION TITLE
Import only maps from collection

### DISCUSSION THREAD

**Discussion by b4mbus88** [ppy/osu#20461] [Ideas] [answered]
It seems like it is not possible to only import maps that are in imported collection. Currently i have a lot of maps in osu!stable, but it is just not useful to import everything. I guess if there would be a way to just import the maps from the collections, alot more players would give lazer a try.

**Comment by peppy** [comment, marked as answer]
Sounds like an interesting option, but I hope to allow for full imports at high speeds (using hard links) which may make this irrelevant.
